src/ServiceManagement/RecoveryServices/Commands.RecoveryServices/RecoveryServicesCmdletBase.cs

```diff
@@ -157,6 +270,16 @@ public void WaitForJobCompletion(string jobId)
         }
 
         /// <summary>
+        /// Writes content to the screen.
+        /// </summary>
+        /// <param name="contents">Data to be printed on the screen.</param>
+        public void WriteResponse(string contents)
+        {
+            this.WriteObject(Environment.NewLine);
```

Comment:
This is not changed as asked.  Please change this to not write to the output stream